tests/k8s_cmdline_tests.yaml

```diff
@@ -156,3 +156,44 @@
     --pod_list k8s_testcases/example_podlist/pods_list.json
     --return_0
   expected: 0
+### adding tests with resources_list combinations:
+- name: basic_sanity_unused_resource_np  # resource_list will be ignored cause there is a np_list with the query arg
+  args: >
+    --sanity k8s_testcases/example_policies/testcase8/testcase8-networkpolicy1.yaml
+    --resource_list k8s_testcases/example_policies/testcase7/testcase7-networkpolicy1.yaml
+    --ns_list k8s_testcases/example_podlist/ns_list.json
+    --pod_list k8s_testcases/example_podlist/pods_list.json
+  expected: 1
+
+- name: basic_sanity_unused_resource_pods  # resource_list will be ignored cause there is a pod_list arg
+  args: >
+    --sanity k8s_testcases/example_policies/testcase8/testcase8-networkpolicy1.yaml
+    --resource_list k8s_testcases/example_podlist/bank_pod_list.json
+    --ns_list k8s_testcases/example_podlist/ns_list.json
+    --pod_list k8s_testcases/example_podlist/pods_list.json
+  expected: 1
+
+#- name: basic_sanity_unused_resource_pods_ns  # resource_list will be ignored cause there are pod_list and ns_list args
+#  args: >
+#    --sanity k8s_testcases/example_policies/testcase8/testcase8-networkpolicy1.yaml
+#    --resource_list k8s_testcases/example_podlist/bank_pod_list.json
+#    --resource_list k8s_testcases/example_podlist/bank_ns_list.json
+#    --ns_list k8s_testcases/example_podlist/ns_list.json
+#    --pod_list k8s_testcases/example_podlist/pods_list.json
+#  expected: 1
+
+- name: basic_sanity_used_resource_np  # the networkpolicy will be read from the resource_list
```

Comment:
Can add a test where the ns_list and pod_list are also replaced by resource_list ? (is it supposed to work on this current version?)

Reply:
yes , I will add it,
I couldn't add a test with separate paths (since action append is not supported yet because of handling the networkpolicies)
But I can attach the dir path of ns and pods to the resource_list , it should work